### PR TITLE
add missing help for top level command group

### PR DESCRIPTION
**Pull request by kairu-ms**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
add missing help

File: src/dnc/azext_dnc/manual/_help.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+helps['dnc'] = """
+    type: group
+    short-summary: Manage Delegated Network
+"""
```
Comment (kairu-ms) [inline src/dnc/azext_dnc/manual/_help.py:12]:
```suggestion
"""

```

File: src/guestconfig/azext_guestconfig/manual/_help.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+helps['guestconfig'] = """
+    type: group
+    short-summary: Manage Guest Configuration
+"""
```
Comment (kairu-ms) [inline src/guestconfig/azext_guestconfig/manual/_help.py:12]:
```suggestion
"""

```

File: src/healthcareapis/azext_healthcareapis/manual/_help.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+helps['healthcareapis'] = """
+    type: group
+    short-summary: Manage Healthcare Apis
+"""
```
Comment (kairu-ms) [inline src/healthcareapis/azext_healthcareapis/manual/_help.py:12]:
```suggestion
"""

```

File: src/logic/azext_logic/manual/_help.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+helps['logic'] = """
+    type: group
+    short-summary: Manage Logic
+"""
```
Comment (kairu-ms) [inline src/logic/azext_logic/manual/_help.py:12]:
```suggestion
"""

```

Comment (kairu-ms) [inline src/logic/azext_logic/manual/_help.py]:
```suggestion
"""

```

File: src/portal/azext_portal/manual/_help.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+helps['portal'] = """
+    type: group
+    short-summary: Manage Portal
+"""
```
Comment (kairu-ms) [inline src/portal/azext_portal/manual/_help.py:12]:
```suggestion
"""

```

File: src/securityinsight/azext_sentinel/manual/_help.py [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+helps['sentinel'] = """
+    type: group
+    short-summary: Manage Security Insight
+"""
```
Comment (kairu-ms) [inline src/securityinsight/azext_sentinel/manual/_help.py:12]:
```suggestion
"""

```

File: src/stack-hci/azext_stack_hci/manual/_help.py [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+helps['stack-hci'] = """
+    type: group
+    short-summary: Manage Azure Stack HCI
+"""
```
Comment (kairu-ms) [inline src/stack-hci/azext_stack_hci/manual/_help.py:12]:
```suggestion
"""

```

**Review by yonzhan — APPROVED** [review]

**Pull request merged**